test/models/user_test.rb

```diff
@@ -63,6 +63,24 @@ def setup
       user.update!(require_2sv: true)
       assert_nil user.reason_for_2sv_exemption
     end
+
+    should "require 2SV for the user when it is switched on for their organisation and they are not exempt" do
+      organisation = create(:organisation, require_2sv: false)
+      user = create(:user, organisation:)
+      assert_not user.require_2sv?
+
+      organisation.update!(require_2sv: true)
+      assert user.require_2sv?
+    end
+
+    should "not require 2SV for the user when it is switched on for their organisation and they are exempt" do
+      organisation = create(:organisation, require_2sv: false)
+      user = create(:user, organisation:, reason_for_2sv_exemption: "User is exempt")
```

Comment:
Just a note - if my PR gets merged in before this, this will start failing as it introduces a validation that if a reason is present, an expiry date is also present. There's a factory for an exempted user in that PR. Only flagging to save you a bit of time debugging because I remember it silently failing on create before and that being confusing 🙃

Reply:
Cool, thanks for the warning.  I'll assume your PR will get merged first, so I'll rebase and update this test once that's done.